lib/manager/github-actions/extract.spec.ts

```diff
@@ -18,5 +18,12 @@ describe('manager/github-actions/extract', () => {
         res.deps.filter((d) => d.datasource === 'github-tags')
       ).toHaveLength(5);
     });
+    it('extracts tag line with double quotes', () => {
+      const res = extractPackageFile(Fixtures.get('workflow_3.yml'));
+      expect(res.deps).toMatchSnapshot();
+      expect(
+        res.deps.filter((d) => d.datasource === 'github-tags')
+      ).toHaveLength(1);
```

Comment:
```suggestion
      expect(res.deps).toMatchSnapshot([
  {
    "currentValue": "v0.13.1",
    "datasource": "github-tags",
    "depName": "pascalgn/automerge-action",
    "depType": "action",
    "replaceString": "pascalgn/automerge-action@v0.13.1",
    "versioning": "docker",
  },
]);
```

some more explict matches